README.md

```diff
@@ -85,14 +85,18 @@ python -m goth start your/output/dir/goth-config.yml
 
 If everything went well you should see the following output:
 ```
-Now run your requestor agent as follows:
+Local goth network ready!
 
-$ PATH=/tmp/...:$PATH YAGNA_APPKEY=3438...7901 YAGNA_API_URL=http://172.19.0.6:6000 GSB_URL=tcp://172.19.0.6:6010 examples/blender/blender.py --subnet goth
+You can now load the requestor configuration variables to your shell:
 
-Press Ctrl+C at any moment to stop the test harness.
+source /tmp/goth_interactive.env
```

Comment:
I didn't raise this objection when we decided to put test logs in `/tmp/goth-tests` so maybe now it's too late to complain about it, but I don't like the idea of putting any files that can be accessed by user -- such as the log files, which are used as test artifacts, or this .env file -- in `/tmp`.

While I'm not proposing to change the location of the log files now, how about writing the .env file in the current working directory?